src/Components/Slides/SlidesForm.js

```diff
@@ -1,32 +1,107 @@
-import React, { useState } from 'react';
-import '../FormStyles.css';
+import axios from "axios";
+import React, { useState } from "react";
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
+import { CKEditor } from '@ckeditor/ckeditor5-react';
+import "../FormStyles.css";
+import Swal from 'sweetalert2'
 
-const SlidesForm = () => {
-    const [initialValues, setInitialValues] = useState({
-        name: '',
-        description: ''
-    });
-
-    const handleChange = (e) => {
-        if(e.target.name === 'name'){
-            setInitialValues({...initialValues, name: e.target.value})
-        } if(e.target.name === 'description'){
-            setInitialValues({...initialValues, description: e.target.value})
+const SlidesForm = ({slides}) => {
+  const [initialValues, setInitialValues] = useState({
+    name: slides?.name ?? '',
+    description:slides?.description ?? '',
+    image:slides?.image ?? '',
+    order:slides?.order ?? '',
+  });
+  const hasSlides = !!slides
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  const handleData = () => {
+    if(!hasSlides){
+    axios({
+      method: "POST",
+      url: "http://ongapi.alkemy.org/api/slides",
+      data: {
+        name: initialValues.name,
+        description: initialValues.description,
+        image: initialValues.image,
+      },
+    })
+      .then((res) => {
+        if (initialValues.name.length>4) {
```

Comment:
this validation is a little bit difficult to read. you are validating if the name of the slide length is bigger than 4 after create the slide so it do not make too much sense. This type of validations should be done before save the value.

Reply:
Done